modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="enabling-encapsulation_{context}"]
+= Working with mount namespace encapsulation
+
+As an OpenShift system administrator, you want `systemd` to consume less resources so that you can run more workloads on my system. You want to enable and disable this setting, and verify whether encapsulation is in effect.
+
+== Enabling and disabling encapsulation
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+
+* You have installed the OpenShift CLI (oc).
+
+* You have created the `MachineConfig` object.
+
+.Procedure
+
+. Add the following YAML to the `MachineConfig.yaml` file to control whether encapsulation is enabled or disabled:
+
++
+[source,yaml]
+----
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: master
+    name: 99-kubens-master
+  spec:
+    config:
+      ignition:
+        version: 2.2.0
```

Comment:
Why such an old version?

Reply:
The ignition version only matters if you're trying to use new features, and this one (enabling a systemd service) is a very old feature.

Updating this to 3.2.0 (or whatever the latest version is today) would not incorrect, but leaving it as-is is also fine (from a technical perspective anyway)

Reply:
Updated to 3.2.0 for consistency.